package.json

```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "watch-server": "nodemon ./server/start.js --ignore public/",
-    "build-js": "parcel build -d server/client/dist server/client/client.js --public-url ./",
-    "build-css": "concurrently \"parcel build -d server/client/dist server/client/normalise.css --public-url ./\" \"parcel build -d server/client/dist server/client/style.css --public-url ./ \" ",
-    "build-html": "parcel build -d server/client/dist server/client/main.html --public-url ./",
+    "build-js": "parcel build server/client/client.js --out-dir server/client/dist  --public-url ./ --log-level 1",
```

Comment:
this reduces the logs being filled with build statements, will only log errors now